### DISCUSSION TITLE
How does font rendering differ from freetype default?

### DISCUSSION THREAD

**Discussion by nlgranger** [kovidgoyal/kitty#6647] [General]
Kitty has a nice font rendering which is close to the gtk one under gnome: crisp with grayscale subpixel mode, font looks slightly bigger than gtk for a given font size.

For comparison, Alacritty and foot look slightly blurrier and darker (bright color on dark background).

Does kitty use tricks or overrides fontconfig settings to make fonts render better?

**Comment by kovidgoyal** [comment]
See https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.text_composition_strategy

and note that kitty does not do subpixel antialiasing only normal
antialiasing.

**Reply by nlgranger** [reply]
Got it thanks! The culprit was actually lcdfilter which was not disabled (`none` instead of `lcdnone`).

No subpixel antialiasing is the way to go in order to avoid color fringing.
